### ISSUE TITLE
sharing files other than images crashes riotx

### ISSUE THREAD

**Issue by lub** [vector-im/riotX-android#987]
Doesn't matter if e2ee or plain. I've sent a rageshake.

steps to reproduce:

1. go to another app (e.g.my music player)
2. share a file
3. select riotx
4. select room
5. (app crashed)

**Comment by bmarty** [comment]
Thanks for reporting the issue and for the rageshake. We will investigate

**Comment by returntrip** [comment]
Does NOT happen for me on `0.15.0 [40015002] (G-b1855)`

**Comment by lub** [comment]
hmm, maybe this is particularly a problem with music files? (e.g. mp3, m4a, ..)

**Comment by returntrip** [comment]
Just tried with a .m4a and that worked. Not sure why does not work for you. Maybe check the app permissions?

**Comment by bmarty** [comment]
A missing permission issue has been fixed in #1010 . This is maybe the root cause of the pb.
But the crash should also occurs for images.

**Comment by bmarty** [comment]
Here are the log from the rageshake:
```
Thread: main, Exception: java.lang.IllegalStateException: firstFrame must not be null
at im.vector.matrix.android.internal.session.room.send.LocalEchoEventFactory.createVideoEvent(LocalEchoEventFactory.kt:238)
at im.vector.matrix.android.internal.session.room.send.LocalEchoEventFactory.createMediaEvent(LocalEchoEventFactory.kt:178)
at im.vector.matrix.android.internal.session.room.send.DefaultSendService.sendMedia(DefaultSendService.kt:192)
```

I will see how to handle that gracefully

**Comment by bmarty** [comment]
Ok, fixed by 4ba8a42977cd7e48623d48f6b4d6b4e8bdabafb0. It should not crash anymore

The route cause is not fixed though. RiotX should not handled audio file has video file.

**Comment by bmarty** [comment]
Thanks for having closed the issue :)
